airflow/models/baseoperator.py

```diff
@@ -497,7 +497,7 @@ def __init__(
         params: Optional[Dict] = None,
         default_args: Optional[Dict] = None,
         priority_weight: int = 1,
-        weight_rule: str = WeightRule.DOWNSTREAM,
+        weight_rule: str = conf.get('core', 'default_task_weight_rule'),
```

Comment:
```suggestion
        weight_rule: str = conf.get('core', 'default_task_weight_rule', fallback=WeightRule.DOWNSTREAM),
```
Should we have a fallback too?

Reply:
added